src/org/zaproxy/zap/extension/saml/ui/SamlManualEditor.java

```diff
@@ -223,7 +223,7 @@ public void focusLost(FocusEvent e) {
      * @param msg
      */
     private void updateResponse(HttpMessage msg) {
-        respBodyTextPane.setText(msg.getResponseBody().createCachedString("UTF-8"));
+        respBodyTextPane.setText(msg.getResponseBody().toString());
```

Comment:
This is to fix the compilation error caused by the update of zap.jar. Although, the update of the JAR shouldn't be needed as we would like to target previous versions of ZAP, not just 2.6.x (any required core enhancements should be bundled in the add-on).